vermouth/rcsu/go_pipeline.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Wrapper of Processors defining the GoPipline.
+"""
+import inspect
+import vermouth
+from ..processors.processor import Processor
+from .go_vs_includes import VirtualSideCreator
+from .go_structure_bias import ComputeStructuralGoBias
+from ..processors import SetMoleculeMeta
+
+class GoProcessorPipline(Processor):
+    """
+    Wrapping all processors for the go model.
+    """
+    def __init__(self, processor_list, **kwargs):
+        self.processor_list = processor_list
+        self.kwargs = kwargs
+
+    def prepare_run(self, system):
+        """
+        Things to do before running the pipeline.
+        """
+        # merge all molecules in the system
+        # this will eventually become deprecated
+        # with the proper Go-model for multimers
+        vermouth.MergeAllMolecules().run_system(system)
```

Comment:
No because the moltype has to be set after this processor is run. I also thought I could make the res_graph here but that breaks because the VS first have to be added see prev. discussion. Once Go can do multiple chains we probably can get rid of it all together.